src/Constants.iOS.cs.in

```diff
@@ -138,5 +137,6 @@ namespace MonoTouch {
 		// iOS 13.0
 		public const string SoundAnalysisLibrary = "/System/Library/Frameworks/SoundAnalysis.framework/SoundAnalysis";
 		public const string IncomingCallNotificationsLibrary = "/System/Library/Frameworks/IncomingCallNotifications.framework/IncomingCallNotifications";
+		public const string SpeechLibrary = "/System/Library/Frameworks/Speech.framework/Speech";
```

Comment:
Drop the change on this file, it is not needed at all.

Once a framework is on that list, you do not need to add it again

Reply:
please add some context if you _resolve_ a comment
right now it's unclear to me why this change should be accepted (since @dalexsoto comment makes sense and is not addressed)

Reply:
Oops sorry, I made the changes but forgot to push them